api/dss.yaml

```diff
@@ -1178,7 +1255,7 @@ components:
         Identification Service Area in the DSS.
       required:
       - extents
-      - flights_url
```

Comment:
flights_url is more specific than 'url'.  There are many URLs and only some of them are for flights -- more specificity here adds clarity.

Reply:
url -> flights_url